README.md

```diff
@@ -82,7 +82,7 @@ _Packages for building GraphQL APIs via ASGI._
 _Packages for monitoring ASGI web applications._
 
 <!-- sort_by:name -->
-
+- [New Relic ASGI](https://docs.newrelic.com/docs/agents/python-agent/python-agent-api/asgiapplication-python-agent-api) - New Relic's API for ASGI web servers and frameworks.
```

Comment:
Should we try and align the presentation here with the Sentry SDL entry?

We'd need to point in which PyPI package this integration can be sound, and update the phrasing to something like...

> New Relic integration for ASGI applications. (Shipped with `<package>`.)

(I know this doesn't match the "ASGI frameworks" phrasing in the Sentry entry, but I don't think that phrasing is ideal, since the integration can be used even on apps that don't fit within a given "framework".)

Reply:
The New Relic integration includes API calls that can be used with New Relic's python agent. The support was shipped with the following release of NR's python agent package: https://pypi.org/project/newrelic/5.16.2.147/.

There is also direct agent support now for Uvicorn Server: https://pypi.org/project/newrelic/5.20.0.149/ and Starlette support is next.

I am interested in your opinion on capturing this. Do you think the following would be clear enough? 

New Relic integration for ASGI applications. (Shipped with [newrelic](https://pypi.org/project/newrelic/))

Reply:
> New Relic integration for ASGI applications. (Shipped with newrelic)

Yup, sounds great! :-)

Reply:
Thank you for your feedback! I made those changes.